tst/EnergyPlus/unit/ExtendedHI.unit.cc

```diff
@@ -0,0 +1,625 @@
+                                                   366.6835501792957,
+                                                   370.2605324515025,
+                                                   373.7949378686608},
+                                                  {329.9497329717269,
+                                                   335.7802855706541,
+                                                   339.9405041738646,
+                                                   344.0269053383963,
+                                                   348.1004405542626,
+                                                   352.130907458195,
+                                                   356.1095161383855,
+                                                   360.03710570017574,
+                                                   363.9145213572192,
+                                                   367.74261204962386,
+                                                   371.5222280813032,
+                                                   375.2542191799148,
+                                                   378.9394325375906},
+                                                  {334.63809705979656,
+                                                   339.06981446722057,
+                                                   343.41864144138526,
+                                                   347.7367934017093,
+                                                   352.013488568773,
+                                                   356.2318584616878,
+                                                   360.3929031532607,
+                                                   364.49763041891856,
+                                                   368.54705243284116,
+                                                   372.5421826945967,
+                                                   376.48403349361615,
+                                                   380.3736132776248,
+                                                   384.2119247702067},
+                                                  {337.80000485421624,
+                                                   342.43363185960334,
+                                                   347.0118012771127,
+                                                   351.55276412755484,
+                                                   356.0280998123926,
+                                                   360.438997685269,
+                                                   364.78665706905304,
+                                                   369.07228278025286,
+                                                   373.2970810952247,
+                                                   377.4622562734294,
+                                                   381.56900731119094,
+                                                   385.6185252717114,
+                                                   389.61199074954493}};
```

Comment:
So where did these values come from?  I am guessing they are just generated from running the code, not from an external source.  If so, this is OK, but not great.  If these values are from a separately validated source, then that is a nice verification.